shipitscript/ship_actions.py

```diff
@@ -12,9 +12,9 @@ def mark_as_shipped(ship_it_instance_config, release_name):
     """
     auth = (ship_it_instance_config['username'], ship_it_instance_config['password'])
     api_root = ship_it_instance_config['api_root']
-    timeout_in_seconds = ship_it_instance_config['timeout_in_seconds']
+    timeout_in_seconds = int(ship_it_instance_config['timeout_in_seconds'])
```

Comment:
This appears to be missing from your config.example.json :)

Reply:
Nice catch! Done. I also made this value optional.